ush/cam/evs_href_prepare.sh

```diff
@@ -166,7 +166,7 @@ if [ "$data" = "ccpa24h" ] ; then
       else
          if [ "$SENDMAIL" = "YES" ] ; then
             export subject="06h CCPA Data Missing for 24h CCPA generation"
-            echo "Warning: At least one of ccpa06h files is missing  for ${VDATE}" > mailmsg
+            echo "WARNING: At least one of ccpa06h files is missing  for ${VDATE}" > mailmsg
             echo -e "`cat $DATA/job${data}${domain}_missing_24hrccpa_list`" >> mailmsg
             echo "Job ID: $jobid" >> mailmsg
```

Comment:
We need to move this outside the SENDMAIL block, like you did below for RAP prepbufr, including the full path.